src/daily_output.ts

```diff
@@ -90,6 +90,17 @@ export class DailyOutput {
 		this.data.blockHeight = this.heightRange[1];
 		this.data.timestamp = this.latestTimestamp;
 
+		this.caclulateChangeOfUploadVolume();
+		this.caclulateWeeklyRewards();
+
+		// compute streak rewards
+		// const stakedPSTHolders = Object.keys(this.data.PSTHolders);
+		// TODO: determine if the wallets has uploaded data for 7 days in a row
+
+		this.data.lastUpdated = Date.now();
+	}
+
+	private caclulateChangeOfUploadVolume(): void {
 		const addresses = Object.keys(this.data.wallets);
 		addresses.forEach((address) => {
```

Comment:
Instead of weekly and last week we are using yesterday and today!

Reply:
[0e4835d](https://github.com/ardriveapp/inferno-backend/pull/29/commits/0e4835de1265bd29aa6c33178dd78ff41f557ddf)